app/views/users/new.html.erb

```diff
@@ -1,6 +1,39 @@
 <%
   title = t('users.new.title')
   provide(:title, title)
+
+  default_input = 'text_field'
+  available_inputs = {
+    :first_name             => { size: 'col-6' },
+    :last_name              => { size: 'col-6' },
+    :email                  => { size: 'col-6' },
+    :phone_number           => { size: 'col-6' },
+    :username               => { size: 'col-12' },
+    :password               => { size: 'col-6', type: 'password' },
+    :password_confirmation  => { size: 'col-6', type: 'password', label: 'Confirmation' }
+  }
```

Comment:
This is pretty non-standard in how we build forms.

Better to do these explicitly usually.

Also, you can use the locale file for your custom label:
```yaml
# en.yml
en:
  activerecord:
    attributes:
      user:
        password_confirmation: "Confirmation" # this will change it in error messages etc
  users:
    new:
      password_confirmation: "Confirmation" # this will just change it in the view label
```
If you change it in the active record attributes, you don't need to specify it in the ERb file; if you use the view scoping, you can use the shorthand `t(".password_confirmation")` for your label definition.